src/assets/i18n/cs.json5

```diff
@@ -2469,6 +2469,28 @@
   "item.edit.withdraw.header": "Vyřazený záznam: {{ id }}",
   // "item.edit.withdraw.success": "The item was withdrawn successfully",
   "item.edit.withdraw.success": "Záznam byl úspěšně vyřazen",
+  // "item.edit.tabs.license.head": "License",
+  "item.edit.tabs.license.head": "Licence",
+  // "item.edit.license.head": "Current License",
+  "item.edit.license.head": "Stávající licence",
+  // "item.edit.license.select.license.initial-value.message": "-Select License-",
+  "item.edit.license.select.license.initial-value.message": "-Vyberte licenci-",
+  // "item.edit.license.select.license.select.message": "Select a license",
+  "item.edit.license.select.license.select.message": "Vyberte licenci",
+  // "item.edit.license.select.license.info": "If you want to change the license, select a new license from the list and press update.",
+  "item.edit.license.select.license.info": "Přejete-li si změnit licenci, vyberte ji ze seznamu a zmáčkněte změnit.",
```

Comment:
zmáčkněte ?

Reply:
klikněte 
ale co je to "zmenit"?

Reply:
I copied all translations from the current Lindat page. Should I change it?

Reply:
do we have a button called "zmenit"?

Reply:
Changed to `zmáčkněte aktualizovat`